uber/templates/registration/reg_take_report.html

```diff
@@ -56,7 +56,7 @@
     {% for sale in sales %}
         <tr>
             <td>{{ sale.what }}</td>
-            <td><a href="../form?id={{ attendee.id }}">{{ sale.attendee.full_name }}</a></td>
+            <td><a href="../registration/form?id={{ sale.attendee_id }}">{{ sale.attendee.full_name }}</a></td>
```

Comment:
Since we're already in the registration subpath, we could use `./form?id=` instead of `../registration/form?id=`.